GameServer/Logic/GameObjects/LevelProp.cs

```diff
@@ -70,6 +72,10 @@ public string getType()
         {
             return type;
         }
+        public byte getSkin()
```

Comment:
Method names with capitalized first letter please. Moreover, I think our style guide has a rule for doing getters/setters with the C# syntax, so the `skin` variable should be defined as
```CS
public byte Skin { get; private set; }
```